html-report/src/components/SearchBar.js

```diff
@@ -1,21 +1,128 @@
 import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import elasticlunr from 'elasticlunr';
 
+const SEARCH_FIELDS = ['package_name', 'class_name', 'name', 'id', 'status'];
+const SEARCH_REF = 'id';
+const EL_SEARCH = elasticlunr();
+const STATUSES = ['failed', 'ignored'];
```

Comment:
lol, this is not what I meant :)

I meant that by clicking on existing `Passed`, `Failed`, `Ignored` blocks search bar should filter tests with clicked status:
<img width="999" alt="screen shot 2017-06-19 at 09 23 25" src="https://user-images.githubusercontent.com/967132/27272080-0df821ce-54d1-11e7-97da-ad9073f13bda.png">